layouts/_default/baseof.html

```diff
@@ -26,7 +26,7 @@
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <img class="navbar-brand" src="https://cdn.earthdata.nasa.gov/eui/latest/docs/assets/ed-logos/meatball_hover.png" alt="Meatball" />
```

Comment:
I'm not referring the CDN hosted PNG file. I'm referring to the SVG file I'd originally established for use in README:
https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/master/css/images/nasa-logo.svg